tests/k8s/test_configmap.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+import mock
+import pytest
+
+from k8s.client import NotFound
+from k8s.models.common import ObjectMeta
+from k8s.models.configmap import ConfigMap
+
+NAME = "my-name"
+NAMESPACE = "my-namespace"
+
+
+@pytest.mark.usefixtures("k8s_config")
+class TestIngress(object):
```

Comment:
Weird name for a test of configmaps...

Reply:
Wonder how that happened. Definitely not because I copied the code from the other test..